spring-data-jpa/src/main/java/org/springframework/data/jpa/repository/query/JpqlQueryRenderer.java

```diff
@@ -2040,9 +2035,7 @@ public List<JpaQueryParsingToken> visitSimple_case_expression(JpqlParser.Simple_
 		tokens.add(new JpaQueryParsingToken(ctx.CASE()));
 		tokens.addAll(visit(ctx.case_operand()));
 
-		ctx.simple_when_clause().forEach(simpleWhenClauseContext -> {
-			tokens.addAll(visit(simpleWhenClauseContext));
-		});
+		ctx.simple_when_clause().forEach(simpleWhenClauseContext -> tokens.addAll(visit(simpleWhenClauseContext)));
 
```

Comment:
I am usually a fan of shortening things like this, but since this is an ANTLR Visitor implementation, I actually prefer to retain it the way it is. It makes it easy when grammar tweaks require me to add another `tokens.addAll(...)` inside that clause. And since this pattern is used A LOT in this class (and its counterpart), it becomes easier to maintain if I retain the pattern consistently everywhere.